tests/default_vars.sh

```diff
@@ -499,6 +499,7 @@ export POSTXCONFIG=postxconfig-NT-gfs.txt
 export POSTXCONFIG_FH00=postxconfig-NT-gfs_FH00.txt
 
 export FV3=true
+export MPAS=true
```

Comment:
I would suggest `export MPAS=false` here in export_fv3 so both aren't true. Making a separate `export_mpas ()` is one option. If you want many of the same settings while turning on MPAS-A, it may be more concise to set:
```
export_fv3
export FV3=false
export MPAS=true
```
in a regression test until there are defaults to set.